.env.in

```diff
@@ -1,7 +1,2 @@
 MONGODB_URI=[Connection String from Azure, how to change it read in Readme, https://github.com/KTH/kurs-pm-data-api/blob/master/README.md#prepara-database-in-azure]
```

Comment:
Everything looks good, I just think if you write this environment variable like this "MONGODB_URI=[Available in Azure KeyVault]" it will help us to have same structure in all of our repos.